internal/pkg/keeper/api/caller.go

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright (C) 2024 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package api
+
+import (
+	"errors"
+
+	"github.com/edgexfoundry/go-mod-configuration/v3/internal/pkg/keeper/utils/http"
+)
+
+type Caller struct {
+	baseUrl string
+}
+
+// NewCaller creates an instance of Caller
+func NewCaller(baseUrl string) *Caller {
+	return &Caller{
+		baseUrl: baseUrl,
+	}
+}
+
+func (c *Caller) Ping() error {
```

Comment:
All public method must have godocs style comments

Reply:
File removed, using the http clients from core-contract.